src/main/java/Duke.java

```diff
@@ -1,10 +1,104 @@
+import java.util.Arrays;
+import java.util.Scanner;
 public class Duke {
     public static void main(String[] args) {
```

Comment:
main() could be too long, maybe try to refactor into new methods

Reply:
You're right, will attempt to make my code more readable by splitting functionality into more methods